cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineConfigurationDTO.java

```diff
@@ -64,6 +77,7 @@ public Integer getRam()
         return ram;
     }
 
+    @Deprecated
     public void setRam( Integer ram )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)